ansible_collections/arista/avd/roles/eos_validate_state/python_modules/constants.py

```diff
@@ -34,6 +35,7 @@
     AvdTestRoutingTable: {"legacy_ansible_tags": ["routing_table"]},
     AvdTestInbandReachability: {"legacy_ansible_tags": ["loopback_reachability", "loopback0_reachability", "optional"]},
     AvdTestLoopback0Reachability: {"legacy_ansible_tags": ["loopback_reachability", "loopback0_reachability"]},
+    AvdTestHttpsSSL: {"legacy_ansible_tags": ["ssl"]},
```

Comment:
There was no legacy tags for this so you can leave it empty
```suggestion
    AvdTestHttpsSSL: {},
```

I would prefer if the above works but check that we are not assuming the key always exist ;)

Reply:
Empty dict is not working, gives key error for legacy_ansible_tags so updated the get_anta_results file for the same